src/CommandLine/.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*.cs]
+
+# Justification: Application project
```

Comment:
The original suppression file had "Synchronization context is known" as the reason, so I'm thinking something more like:

```suggestion
# Justification: App synchronization contexts don't require ConfigureAwait(false)
```